src/ChildWorkflowStub.php

```diff
@@ -31,8 +32,30 @@ public static function make($workflow, ...$arguments): PromiseInterface
             return resolve(Y::unserialize($log->result));
         }
 
-        WorkflowStub::make($workflow)
-            ->startAsChild($context->storedWorkflow, $context->index, $context->now, ...$arguments);
+        $childWorkflowRunning = false;
+
+        $storedChildWorkflow = $context->storedWorkflow->children()
+            ->wherePivot('parent_index', $context->index)
+            ->first();
+
+        if ($storedChildWorkflow) {
+            $childWorkflow = $storedChildWorkflow->toWorkflow();
+            if ($childWorkflow->running()) {
+                try {
+                    $childWorkflow->resume();
+                } catch (\Spatie\ModelStates\Exceptions\TransitionNotFound) {
+                }
+                $childWorkflowRunning = true;
+            } elseif ($childWorkflow->status() === WorkflowFailedStatus::class) {
```

Comment:
@rmcdaniel should this follow the helper method pattern of `running` and `resume to be `$childWorkflow->failed()`?